### PR TITLE
Follow up changes on rebranding

### PR DESCRIPTION
**Pull request by ddouglasz**
#### Summary

Follow up changes on recolouring work. 
Icon color on project selection appbar component and update to the global notification background color for warning.

### CODE REVIEW

**Review by CarlosCortizasCT — APPROVED** [review]
Thanks! 👍

File: .changeset/good-pigs-sniff.md
Comment (ddouglasz) [reply .changeset/good-pigs-sniff.md]: updated here: https://github.com/commercetools/merchant-center-application-kit/pull/3440/commits/c2072ce4696648e7724227fb764eb01075aaa154

Thank you

**Review by kark — APPROVED** [review]

**Review by tdeekens — APPROVED** [review]

**Comment by CarlosCortizasCT** [comment]
I see we already merges this PR but we don't have @FilPob approval yet.

Maybe for next UI related PRs, I think it would be great if we wait for his approval before merging.

**Pull request merged**
